Components/Button/button.css

```diff
@@ -0,0 +1,70 @@
+.btn {
+    font-weight: 600;
+    background: var(--default-color);
+    color: var(--color-white);
+    min-width: 100px;
+    width: max-content;
+    padding: .8rem 1rem;
+    border-radius: 1px;
+    border: none;
```

Comment:
Instead of border: none, you might want to use border transparent
For better accessibility practices